src/JointPositionGoalController.cpp

```diff
@@ -79,10 +83,11 @@ namespace valkyrie_translator {
 
         void publishCommandFeedbackToLCM(int64_t utime);
 
-        boost::shared_ptr<lcm::LCM> lcm_;
+        std::shared_ptr<lcm::LCM> lcm_;
         std::shared_ptr<JointPositionGoalController_LCMHandler> handler_;
 
         std::vector<std::string> joint_names_;
+        std::vector<std::string> joint_state_joint_names_;
```

Comment:
It's for the JointStatePublisher you've been asking for and accidentally snuck in when changing branches (it's working now alongside other controllers, independent of which controllers these are).